llvm/test/CodeGen/X86/vector-compress-avx2.ll

```diff
@@ -0,0 +1,281 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=x86_64 -mattr=avx2 < %s | FileCheck %s
```

Comment:
Why do you have separate avx2 / avx512 test files? Just put both the RUN lines in a single file and use "--check-prefixes=AVX,AVX2" and "--check-prefixes=AVX,AVX512"

Reply:
I split the files because some of the test cases for AVX512 generate really bad code if there are no AVX512 instructions. For example, `@test_compress_large()` with `<64 x i32>` generates ~500 lines of assembly. If I keep both runs in a single file and generate all tests for non-AVX512, the test file is ~5000 lines. The logic for non-AVX512 is also covered in the AArch64 tests, as it is the same expand code. So I'm not sure how valuable they are. But I'm happy to do this if you think it makes sense.